concrete/single_pages/dashboard/system/registration/automated_logout.php

```diff
@@ -0,0 +1,101 @@
+<?php
+defined('C5_EXECUTE') or die('Access Denied.');
+
+/* @var Concrete\Core\Form\Service\Form $form */
+/* @var Concrete\Core\Validation\CSRF\Token $token */
+/* @var Concrete\Core\Page\View\PageView $view */
+
+/* @var bool $invalidateOnIPMismatch */
+/* @var bool $invalidateOnUserAgentMismatch */
+/* @var string $trustedProxyUrl */
+/* @var string $invalidateAction */
+/* @var string $saveAction */
+?>
+<form action="<?= $saveAction ?>" method="POST">
+    <?php $token->output('save_automated_logout') ?>
+
+    <fieldset>
+        <legend><?= t('Session Security') ?></legend>
+        <div class="form-group">
+
+            <div class="help-block">
+                <?= t('These settings help prevent a user from stealing other logged in user sessions. You may want to configure %s"Trusted Proxies"%s instead', '<a href="' . $trustedProxyUrl . '">', '</a>') ?>
+            </div>
+            <div class="checkbox">
+                <label>
+                    <?= $form->checkbox('invalidateOnIPMismatch', '1', $invalidateOnIPMismatch) ?>
+                    <?= t('Log users out if their IP changes') ?>
+            </div>
+            <div class="checkbox">
+                <label>
+                    <?= $form->checkbox('invalidateOnUserAgentMismatch', '1', $invalidateOnUserAgentMismatch) ?>
+                    <?= t('Log users out if their browser\'s user agent changes') ?>
```

Comment:
Often these kind of messages are searched for in the code / IDE. Do you see any downside in using `t("Log users out if their browser's user agent changes")` to make sure the string is easy to find?

Reply:
The only downside is that my IDE will complain that I'm using double quotes, do you want to open a PR for this?